release/publish.sh

```diff
@@ -66,6 +67,6 @@ go run main.go publish --release "${WORK_DIR}" \
 # Docker hub doesn't support Helm registries, so we also push these only to GCR.
 go run main.go publish --release "${WORK_DIR}" \
     --cosignkey "${COSIGN_KEY:-}" \
-    --helmhub "gcr.io/istio-release/charts" \
-    --dockerhub "gcr.io/istio-release" \
+    --helmhub "gcr.io/${HELM_BUCKET}" \
```

Comment:
can you make this its own var? The fact the HELM_BUCKET _happens_ to also be the gcr.io relative URL is purely coincidence